src/passthrough/async_io.rs

```diff
@@ -221,8 +221,9 @@ impl<D: AsyncDrive> PassthroughFs<D> {
             e
         })?;
 
-        if let Some(h) = handle {
-            let hd = self.handle_map.get(h, inode)?;
+        if self.no_open.load(Ordering::Relaxed) && handle.is_some() {
```

Comment:
Does this work `if self.no_open.load(Ordering::Relaxed) && let Some(hdl) = handle.as_ref() {`?

Reply:
ok, I'm sensitive to the `unwrap()` below, it's safe though.